.drone.yml

```diff
@@ -902,6 +902,270 @@ depends_on:
+
+depends_on:
+- default-linux-amd64
+- default-windows-1809
+---
+kind: pipeline
+name: docker-image-digests-windows-1903
+
+platform:
+  os: windows
+  arch: amd64
+  version: 1903
+
+steps:
+- name: docker-image-digests
+  image: rancher/drone-docker-image-digests:v0.0.11
+  environment:
+    PLUGIN_GITHUB_REPOSITORY: "rancher/rancher"
+    PLUGIN_GITHUB_TOKEN:
+      from_secret: github_token
+    PLUGIN_GITHUB_TAG: "${DRONE_TAG}"
+    PLUGIN_INPUT_FILE: "rancher-windows-images.txt"
+    PLUGIN_OUTPUT_FILE: "rancher-images-digests-windows-1903.txt"
+    PLUGIN_REGISTRY: "docker.io"
+  volumes:
+    - name: docker_pipe
+      path: \\\\.\\pipe\\docker_engine
+  when:
+    instance:
+      include:
+      - drone-publish.rancher.io
+    event:
+    - tag
+
+volumes:
+  - name: docker_pipe
+    host:
+      path: \\\\.\\pipe\\docker_engine
+
+trigger:
+  event:
+    exclude:
+    - promote
+
+depends_on:
+- default-linux-amd64
+- default-windows-1903
+---
+kind: pipeline
+name: docker-image-digests-windows-1909
+
+platform:
+  os: windows
+  arch: amd64
+  version: 1909
+
+steps:
+- name: docker-image-digests
+  image: rancher/drone-docker-image-digests:v0.0.11
+  environment:
+    PLUGIN_GITHUB_REPOSITORY: "rancher/rancher"
+    PLUGIN_GITHUB_TOKEN:
+      from_secret: github_token
+    PLUGIN_GITHUB_TAG: "${DRONE_TAG}"
+    PLUGIN_INPUT_FILE: "rancher-windows-images.txt"
+    PLUGIN_OUTPUT_FILE: "rancher-images-digests-windows-1909.txt"
+    PLUGIN_REGISTRY: "docker.io"
+  volumes:
+    - name: docker_pipe
+      path: \\\\.\\pipe\\docker_engine
+  when:
+    instance:
+      include:
+      - drone-publish.rancher.io
+    event:
+    - tag
+
+volumes:
+  - name: docker_pipe
+    host:
+      path: \\\\.\\pipe\\docker_engine
+
+trigger:
+  event:
+    exclude:
+    - promote
+
+depends_on:
+- default-linux-amd64
+- default-windows-1909
+---
+kind: pipeline
+name: docker-image-digests-windows-2004
+
+platform:
+  os: windows
+  arch: amd64
+  version: 2004
+
+# remove this and use upstream images when https://github.com/drone/drone-git/pull/25 is merged
+clone:
+  disable: true
+
+steps:
+- name: clone
+  image: luthermonson/drone-git:windows-2004-amd64
+  settings:
+    depth: 20
+
+- name: docker-image-digests
+  image: rancher/drone-docker-image-digests:v0.0.11
+  environment:
+    PLUGIN_GITHUB_REPOSITORY: "rancher/rancher"
+    PLUGIN_GITHUB_TOKEN:
+      from_secret: github_token
+    PLUGIN_GITHUB_TAG: "${DRONE_TAG}"
+    PLUGIN_INPUT_FILE: "rancher-windows-images.txt"
+    PLUGIN_OUTPUT_FILE: "rancher-images-digests-windows-2004.txt"
+    PLUGIN_REGISTRY: "docker.io"
+  volumes:
+    - name: docker_pipe
+      path: \\\\.\\pipe\\docker_engine
+  when:
+    instance:
+      include:
+      - drone-publish.rancher.io
+    event:
+    - tag
+
+volumes:
+  - name: docker_pipe
+    host:
+      path: \\\\.\\pipe\\docker_engine
+
+trigger:
+  event:
+    exclude:
+    - promote
+
+depends_on:
+- default-linux-amd64
+#- default-windows-2004
```

Comment:
why not drop it instead of commenting? Do we want to enable it later?

Reply:
The whole 2004 build pipeline is commented out so I added it so it doesn't get forgotten when its uncommented.